### DISCUSSION TITLE
Use BlueRetro as sender instead of receiver

### DISCUSSION THREAD

**Discussion by joselcl98** [darthcloud/BlueRetro#421] [Ideas]
Hello, I don't know if it could or will be another project that can, but I was going to make a hub for gamecube that would join the four controls with their original connector and connect to the gamecube by type c, it's a mini gamecube project that I'm doing little by little, but I was wondering if you could put an esp32 and a battery in that hub and you could invert the Blueretro function, read the data pin and send it to the console's esp32, or convert it to a generic standard so that other hubs of other consoles can be made and it is not necessary to modify the controller itself, damaging it along the way.

**Comment by rodrigo-guim** [comment]
Not so much information yet, but there is this discussion here: https://github.com/darthcloud/BlueRetro/discussions/349
